public/components/PaginationLinks.js

```diff
@@ -0,0 +1,11 @@
+export default {
+    props: ['links'],
+    methods: {
+        changePage(url) {
+            this.$emit('page-changed', url);
```

Comment:
maybe split the query from the url, page-changed should emit the new page value not really the link